pylint_odoo/checkers/no_modules.py

```diff
@@ -371,11 +377,21 @@ def colon_list_to_dict(self, colon_list):
         """
         return dict([item.split(":") for item in colon_list])
 
+    def replace_and_first_lower(self, string):
+        """ Conver 'Hello World' to 'hello_World'
+
+            :param string: String to convert
+            :returns: string
```

Comment:
Returns docstring don't should be the type of return.
Use a description of that contains this variable